web/console/src/api/websockets.ts

```diff
@@ -1,17 +1,31 @@
 // Copyright (C) 2021 Creditor Corp. Group.
 // See LICENSE for copying information.
 
+import { APIClient } from '.';
+
 /**
  * QueueClient is a ws implementation of users API.
  * Exposes queue-related functionality.
  */
-export class QueueClient {
+export class WebSocketClient extends APIClient {
```

Comment:
would it better to name it ak MatchesClient?
So and in matches client will be ws and htpp implementation. Or in this client we have not only matches domain entities?

Reply:
It would be a game websockets too